doc/source/whatsnew/v0.15.1.rst

```diff
@@ -28,7 +22,7 @@ API changes
 
   .. ipython:: python
 
-     s = Series(date_range('20130101',periods=5,freq='D'))
+     s = pd.Series(pd.date_range('20130101', periods=5, freq='D'))
      s.iloc[2] = np.nan
      s
 
```

Comment:
can you try to redefine `df` and `ts` before line 64 (can't comment there, as it's not edited in the PR)

I think they should be accessible from the previous block. You can also use `# noqa F821` in the failing line, to stop the error from being raised.